zips/draft-ecc-community-and-coinholder.md

```diff
@@ -0,0 +1,188 @@
+
+The terms "Zcash Community Grants" (or "ZCG"), "ZCG Committee", "Financial Privacy Foundation" (or "FPF"), and "Deferred Dev Fund Lockbox" in this document are to be interpreted as defined in ZIP 1015 [^zip-1015].
+
+"Shielded Labs" refers to the Assocation of that name registered in the Swiss canton of Zug under the Unique Identifier CHE-243.302.798.
+
+Protocol-defined Ecosystem Funding is funding from sources defined by the Zcash Protocol for development of the Zcash ecosystem. At the time of writing, Protocol-defined Ecosystem Funding is allocated from a portion of block rewards via Funding Streams [^zip-0207] and Lockbox Funding Streams [^zip-2001].
+
+“ZEC” refers to the native currency of Zcash on Mainnet.
+
+
+# Abstract
+
+This proposal outlines a funding model that gives the community and coin holders distinct voices in determining what, if any, grants are provided to support Zcash’s development and community efforts.
+
+In this model:
+
+* 8% of the block rewards will be allocated to the ZCG for grants by and for the Zcash Community.
+* 12% of the block rewards will accrue to a fund controlled by decisions of coin holders, seeded by the Deferred Dev Fund Lockbox.
+
+The coin-holder-controlled funds may be used to distribute larger grants to ecosystem participants, or left at rest.
+
+This model would be activated for 3 years, allowing enough time to determine whether it should be changed or codified for longer.
+
+
+# Motivation
+
+If no action is taken, in November 2025 funds from block subsidies will stop being directed to Zcash Community Grants [^zip-1015-zcg] and to the Deferred Dev Fund Lockbox established by ZIP 2001 [^zip-2001]. If the block subsidies stop, it will risk a gap in funding of Zcash development organisations, either via ZCG grants or via potential future disbursements from the Deferred Dev Fund Lockbox.
+
+This proposal aims to:
+* decentralize decision-making,
+* hold stakeholders accountable for outcomes,
+* be dynamic enough to allow for change, and
+* provide clarity on decision-making.
+
+It would immediately increase coin-holders’ voice, minimize governance confusion, and simplify decision-making.
+
+
+# Requirements
+
+* There is a well-defined, publically agreed, process for evaluation and community feedback on grant proposals.
+* Funds are held in a multisig resistant to compromise of some of the parties' keys, up to a given threshold.
+* No single party's non-cooperation or loss of keys is able to cause any Protocol-defined Ecosystem Funding to be locked up unrecoverably.
+* During the period of this proposal, the block rewards must be distributed as described in the [Abstract] above.
+* The funds from the 8% and 12% funding streams will be usable immediately.
+* The funds in the Deferred Dev Fund Lockbox will be usable immediately.
+* Any use of Deferred Dev Fund Lockbox funds is consistent with the purpose specified in [^zip-1015-lockbox] of "funding grants to ecosystem participants".
```

Comment:
```suggestion
* Any use of Deferred Dev Fund Lockbox funds must be consistent with the purpose specified in [^zip-1015-lockbox] of "funding grants to ecosystem participants".
```

Reply:
The other requirements use "is", "will be", etc. They should all be changed or none of them changed. I think it reads better as-is.